mlflow/sklearn/utils.py

```diff
@@ -126,16 +133,38 @@ def _get_metrics_value_dict(metrics_list):
         try:
             metric_value = metric.function(**metric.arguments)
         except Exception as e:  # pylint: disable=broad-except
-            _log_warning_for_metrics(metric.name, metric.function, e)
+            _log_warning_for_metrics_or_artifacts(metric.name, metric.function, e, True)
         else:
             metric_value_dict[metric.name] = metric_value
-
     return metric_value_dict
 
 
+# Remove duplicates from y_true while still preserving orders
+def _get_labels(y_true):
+    seen = set()
+    return [x for x in y_true if not (x in seen or seen.add(x))]
+
+
+def _get_artifacts_list(artifacts_list):
+    artifacts_dir = tempfile.mkdtemp()
+    for artifact in artifacts_list:
+        try:
+            # print(_get_arg_names(artifact.function))
```

Comment:
Can we remove extraneous `print()` statements? (e.g., this one and the ones on lines 158-159).